tests/unit_tests/test_wave_06.py

```diff
@@ -143,16 +137,13 @@ def test_swap_best_by_category_reordered():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
-    # Assertions should check:
-    # - That result is truthy
-    # - That tai and jesse's inventories are the correct length
-    # - That all the correct items are in tai and jesse's inventories, and that the items that were swapped are not there
-
-@pytest.mark.skip
+    assert result == True
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert item_f in tai.inventory
+    assert item_c in jesse.inventory
```

Comment:
I'd like to see this unit test confirm that the 2 vendor's inventories weren't changed beyond just swapping the best item by category. What other assertion statements would you write to check that?